src/uu/cksum/src/cksum.rs

```diff
@@ -123,7 +151,12 @@ fn cksum<'a, I>(mut options: Options, files: I) -> UResult<()>
 where
     I: Iterator<Item = &'a OsStr>,
 {
-    for filename in files {
+    let files_vec: Vec<_> = files.collect();
```

Comment:
I would simply use `files` as variable name, there is no need to add its type to the name.